### ISSUE TITLE
Testing Interactivity on a Dev kit

### ISSUE THREAD

**Issue by nitinvenu85** [mixer/interactive-unity-plugin#76]
We are trying to test interactivity on a xbox one dev kit but it looks like the interactivity is never enabled. We are calling MixerInteractive.GoInteractive() in code and on the browser we see a "Loading Interactive controls" thing popup when viewing the channel but the controls never load in completely. Is there some setup that is required before we can test on the dev kits ?

**Comment by payzer** [comment]
Have you checked out this page: https://github.com/mixer/interactive-unity-plugin/wiki/Troubleshooting-guide

Take a look at the section: Troubleshooting Xbox One

There are steps you want to take that are a bit different than testing interactive from either the editor or desktop. Does it work for you in the editor?

**Comment by nitinvenu85** [comment]
Hello,

Yes. Interactivity on the PC is fine its just the devkit that is giving us issues. Could be a bigger issue because when we are broadcasting from the devkit we cant view the stream in the browser. It always says the user is offline. We confirmed that the title id and server configuration are fine.

**Comment by payzer** [comment]
The most common issue is the Xbox test account on the Xbox doesn't have access to the interactive project. A share code is usually the solution in this case. You can get more info about how to do that here: https://github.com/mixer/interactive-unity-plugin/wiki/Using-Share-Codes

Let me know if that doesn't solve your broadcasting issue.

Note: We have also seen another issue where if you broadcast from the guide there is an error, that is different. If you are hitting that also let us know.

**Comment by nitinvenu85** [comment]
So our interactive app is already published and being used by the PC version so i am guessing we don't need shared codes.  We confirmed that mixer status in the code goes to InteractivityEnabled so i think everything is good on the game end but we still only see "Connecting to Interactive" when we are in the streamers channel.

**Comment by payzer** [comment]
I take it that since you've published, you have my email. Could you email me at my Microsoft email & we could coordinate me debugging it. I'll quite good at Xbox One debugging :)

**Comment by payzer** [comment]
We debugged. For others who run into this issue, you need to be logged into mixer.com with the same account as you as broadcasting with on your devkit. There's more info for how to do this here:
https://github.com/mixer/interactive-unity-plugin/wiki/Mixer-and-Xbox-Live
